client/blocks/like-button/button.jsx

```diff
@@ -106,11 +106,12 @@ class LikeButton extends PureComponent {
 			</span>
 		);
 
+		const href = ! isLink ? null : `/stats/post/${ postId }/${ slug }`;
```

Comment:
i think it reads more clearly without the negation:
`const href = isLink ? ...stats... : null;`